src/Executor/Executor.php

```diff
@@ -249,6 +249,31 @@ public function createExecution(
         return $response['body'];
     }
 
+    public function createCommand(
```

Comment:
where do we use this? is it related to detection?

Reply:
yes, in builds worker.
1. Start build container, remove=false
2. Execute command in container to get list of output files
3. Detect SSR or static based on files in output directory